packages/fxa-auth-server/package.json

```diff
@@ -130,7 +130,7 @@
     "safe-regex": "^2.1.1",
     "safe-url-assembler": "1.3.5",
     "sandboxed-regexp": "^0.3.0",
-    "stripe": "^11.12.0",
+    "stripe": "^15.4.0",
```

Comment:
Since these are all the same versions, does it cause anything to break if we remove these from the subpackage dependency lists?

Reply:
Not that I can think of. Should be good to go to remove from the subpackages. 👍